src/components/dashboard/Settings.js

```diff
@@ -291,13 +255,13 @@ const Settings = ({ screenProps, styles, theme, navigation }) => {
               <Section.Row style={styles.languageRow}>
                 <View style={styles.languageInputContainer}>
                   <ModalDropdown
-                    defaultValue={languageCustomLabels[countryCode] ?? t`Select a language...`}
-                    options={[isWeb ? '' : 'DD', ...supportedCountryCodes]} // empty string breaks on native
+                    defaultValue={languageLabels[countryCode] ?? t`Select a language...`}
+                    options={[isWeb ? '' : 'DD', ...countryCodes]} // empty string breaks on native
```

Comment:
why we still using this 'DD' ? empty value should mean 'system default'.

Reply:
We removed it but I had to turn it back here
the component is not able to handle a empty string when rendered on android (as comment points out)